From 82845c75f87cde61899d7526551934b3c3d634ff Mon Sep 17 00:00:00 2001
From: Cody Peter Mello <cody.mello@joyent.com>
Date: Sat, 17 Mar 2018 00:36:47 +0000
Subject: [PATCH] TRITON-75 Allow firewall rules to be applied in order of
 priority TRITON-77 Add support for ESP and AH protocols in firewall rules

---
 config/sapi/services/fwapi/service.json | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/config/sapi/services/fwapi/service.json b/config/sapi/services/fwapi/service.json
index cf220ff1..0fcddcdf 100644
--- a/config/sapi/services/fwapi/service.json
+++ b/config/sapi/services/fwapi/service.json
@@ -17,9 +17,9 @@
         ]
     },
     "metadata": {
-        "FWRULE_VERSION": 3,
+        "FWRULE_VERSION": 4,
         "SERVICE_NAME": "fwapi",
         "FWAPI_LOG_LEVEL": "info"
     },
     "manifests": {}
-}
\ No newline at end of file
+}
-- 
2.21.0

